site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/model/Site.java

```diff
@@ -67,11 +67,18 @@ public Site(final String siteId, String name, String alias, String parentSiteId,
         this.siteId = siteId;
     }
 
+    public Site(final String siteId, String name, String alias, String parentSiteId, String siteType, Address address) {
```

Comment:
We've already get too many Site constructors, just use setters instead and you can take this out

Reply:
Removed constructor and added setter.